chemqulacs/vqe/vqeci.py

```diff
@@ -320,6 +320,24 @@ def vqe(init_params, cost_fn, grad_fn, optimizer):
     return opt_state
 
 
+def _generate_inital_states(
+    n_electron, n_orbitals, excitation_number=0, fermion_qubit_mapping=jordan_wigner
```

Comment:
In PySCF, it's typical for arguments to be ordered as 'number of orbitals, number of electrons,' like in mcscf.CASCI(mf, ncas, nelecas). Since I've pointed out several other changes to make, shall we also reverse the order of n_electrons and n_orbitals for consistency?